src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -26,26 +26,35 @@
 import java.io.IOException;
 import java.time.ZonedDateTime;
 import java.util.regex.Pattern;
+import org.cactoos.scalar.IoCheckedScalar;
 import org.cactoos.scalar.ItemAt;
+import org.cactoos.text.FormattedText;
 import org.cactoos.text.SplitText;
 import org.cactoos.text.TextOf;
+import org.cactoos.text.UncheckedText;
 
 /**
  * RtTransaction.
  *
  * @since 0.1
+ * @checkstyle MagicNumber (500 lines)
```

Comment:
@carlosmiranda see https://github.com/zold-io/java-api/pull/45/files#r205379317

Reply:
@victornoel https://github.com/zold-io/java-api/pull/45#discussion_r205379694 cc @llorllale .

Let's settle on this question, we've spent way too much time on these three PRs.

Reply:
@victornoel @carlosmiranda I've answered [here](https://github.com/zold-io/java-api/pull/45#discussion_r205943989)